server/game/cards/09.4-TCoH/Truthseeker.js

```diff
@@ -0,0 +1,96 @@
+import { Locations, TargetModes } from '../../Constants.js';
+const DrawCard = require('../../drawcard.js');
+
+class Truthseeker extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Look at top 3 cards',
+            when: {
+                onCharacterEntersPlay: (event, context) =>
+                    event.card === context.source &&
+                    [Locations.ProvinceOne, Locations.ProvinceTwo, Locations.ProvinceThree, Locations.ProvinceFour].includes(event.originalLocation)
```

Comment:
Have I missed something?  I don't think it needs to enter play from a province, just enter play.